client/src/ui/BitbakeTaskProvider.ts

```diff
@@ -0,0 +1,131 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) 2023 Savoir-faire Linux. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import type * as child_process from 'child_process'
+import * as vscode from 'vscode'
+import { type BitbakeDriver } from '../driver/BitbakeDriver'
+
+const endOfLine: string = '\r\n'
```

Comment:
Not sure if it is going to have the same behaviour in Windows since it mainly uses CRLF. I will leave it to your discretion

Reply:
Other extensions use \r\n as well. Bitbake can only be run on Linux so the input should always have \n alone. This should be portable but I haven't tested on Windows.